networks/2_get_network_communities.py

```diff
@@ -64,40 +57,44 @@
 # A cluster is a set of core samples that can be built by recursively taking a core sample, finding all of its neighbors that are core samples, finding all of their neighbors that are core samples, and so on. A cluster also has a set of non-core samples, which are samples that are neighbors of a core sample in the cluster but are not themselves core samples. Intuitively, these samples are on the fringes of a cluster.
 #
 # * We define a core sample as being a sample in the dataset such that there exist `min_samples` other samples within a distance of `eps`, which are defined as neighbors of the core sample.
-# * Using all default values
+# * Here we use `eps=8` based on the observations in the [prevous notebook](1_correlation_analysis.ipynb)
 #
 
-# +
 # Clustering using DBSCAN
-# pao1_clustering = DBSCAN().fit(pao1_corr)
-# pa14_clustering = DBSCAN().fit(pa14_corr)
-# -
+if cluster_method == "dbscan":
+    pao1_clustering = DBSCAN(eps=8).fit(pao1_corr)
+    pa14_clustering = DBSCAN(eps=8).fit(pa14_corr)
```

Comment:
This is a minor and very personal style comment: I prefer to move all settings/parameters (like `eps` here or `n_clusters` and `distance_threshold` for hierarchical clustering) to the top of the notebook, like in your "Set user parameters" section. This way all parameters that can be changed are in one place and are not lost in the middle of the code.